### ISSUE TITLE
Assistant crashes on connect if IPv6 is disabled on Windows

### ISSUE THREAD

**Issue by AbortingMission** [Ylianst/MeshCentral#2741]
v0.8.57

With IPv6 disabled, when clicking "Request Help" button, the assistant exe will crash.

Enabling IPv6 solves the issue.

```
DEBUG LOG:
UnhandledExceptionEventSink: System.Net.NetworkInformation.NetworkInformationException (0x80004005): The requested protocol has not been configured into the system, or no implementation for it exists
   at System.Net.NetworkInformation.SystemIPInterfaceProperties.GetIPv6Properties()
   at MeshAssistant.MeshCentralAgent.getNetworkInfo()
```

**Comment by Ylianst** [comment]
Nice catch!!

**Comment by Ylianst** [comment]
Just put in a fix for this, will be in MeshCentral v0.8.58.
